esp/scm/ws_dfu.ecm

```diff
@@ -936,7 +938,9 @@ ESPrequest [nil_remove] DFUFileCreateV2Request
     string RequestId;                   // optional 
     int ExpirySeconds(60);              // how long the retrieved meta info is valid for, to access physical files (in dafilesrv) 
     bool ReturnTextResponse(false);     // Returns response in full not just in blob
-    bool Compressed(false);             // The new file will be compressed on disk    
+    bool Compressed(false);             // The new file will be compressed on disk
+    [min_ver("1.51")] int64 SessionId(0); // session id of this client request    
+    [min_ver("1.51")] int LockTimeoutMs(300); // Max seconds to block waiting for lock    
```

Comment:
this default lock timeout seems small just checking that it is intentional.

Reply:
yes, default should be consistent and same as above, will fix.